deduplicator/src/lib/db_hashes.rs

```diff
@@ -0,0 +1,551 @@
+//! Definitions of the structure of a working database to save addresses and how to interact with
+//! it. This database will be used to save hashes of all imported addresses and compute collisions
+//! between them.
+
+use std::convert::TryInto;
+use std::path::PathBuf;
+
+use rusqlite::{Connection, Statement, ToSql, Transaction, NO_PARAMS};
+use tools::Address;
+
+/// Name of the table containing addresses.
+const TABLE_ADDRESSES: &str = "addresses";
+
+/// Name of the table containing hashes. For each address, multiple hashes may be stored.
+const TABLE_HASHES: &str = "_addresses_hashes";
+
+/// Name of the table listing addresses that have to be removed to eliminate all duplicates.
+const TABLE_TO_DELETE: &str = "_to_delete";
+
+//  ____  _     _   _           _
+// |  _ \| |__ | | | | __ _ ___| |__   ___  ___
```

Comment:
You should really stop with that. It's not nice to read. :-/ (but at least it's a nice separation)

Reply:
Another problem with this is that other developers may not be able to update them :see_no_evil:. 
  So they will probably be lost on the next rename / refactoring.

Reply:
There should be none remaining now... Sorry for this, I was mostly experimenting on convenient ways to add separators ^^